src/scripts.js

```diff
@@ -343,14 +345,14 @@ function displayIngredientDropDown() {
     selectIngredient.innerHTML = `<option value="Choose Ingredient">Choose Ingredient...</option>`
     sortedIngredients.forEach(ingredient => {
         selectIngredient.innerHTML += `
-        <option value="Choose Ingredient">${ingredient.name}</option>`
+        <option value="${ingredient.name}">${ingredient.name}</option>`
     })
 }
 
 function addOrRemoveToPantry(user) {
     pantryTable.innerHTML = ''
     const amount = apiIngredients.reduce((acc, value) => {
-        user.pantry.forEach(current => {
+        user.pantry.pantryData.forEach(current => {
             if(value.id === current.ingredient) {
             var object = {['Ingredient']: value.name, ['Amount']: current.amount}
```

Comment:
amazing logic here. Maybe in the future we could rename object to inventoryItem or pantryItem. I know the instructors get lil mad when we name objects 'objects'

Reply:
I like Kiko's idea of renaming the variable to pantryItem. We could always make a note of this for refactor, and do a find, replace all!

Reply:
Oh yeah, I'll take that of that